src/api.d.ts

```diff
@@ -157,16 +157,14 @@ export interface AbstractAzExtTreeItem {
     isAncestorOfImpl?(contextValue: string | RegExp): boolean;
 }
 
-export type ResolvedAppResourceTreeItemBase = Partial<{ [P in keyof SealedAzExtTreeItem]: never }> & AbstractAzExtTreeItem;
+export type ResolvedAppResourceBase = Partial<{ [P in keyof SealedAzExtTreeItem]: never }> & Partial<AbstractAzExtTreeItem> & { contextValue?: never, contextValues?: string[] };
```

Comment:
Should contextValues be a `string[]`?  In our implementation, we're using the `Set<string>`

Reply:
Hmm... I am normally a fan of modern ES features like Set but I been on the other side and dealt with having to polyfill these features with core-js and webpack is a nightmare. I'm not sure if this is a problem in the vscode extension ecosystem though.

I agree that renaming it to `contextValuesToAdd` (or something similar) would be good.

Reply:
Ok, I renamed it to `contextValuesToAdd`. I also created an interface, `ContextValuesToAdd`, instead of the inline declaration--this lets us have documentation attached so intellisense can help users know what to do. I'm not sold on the interface name `ContextValuesToAdd` so if you have ideas I'm all ears.